aiidalab/app.py

```diff
@@ -228,12 +238,29 @@ def find_matching_releases(self, specifier, prereleases=None):
         matching_releases.sort(key=parse, reverse=True)
         return matching_releases
 
+    @staticmethod
+    def _strict_dependencies_met(requirements, python_bin):
+        """Check whether the given requirements are compatible with the core dependencies of a package."""
+        packages = find_installed_packages(python_bin)
```

Comment:
Please make sure we use canonical names all throughout this function.

Reply:
Yes, I did this in new commit.